dbms/src/test/java/org/polypheny/db/sql/view/ComplexViewTest.java

```diff
@@ -1717,46 +1513,49 @@ public void testQ14() throws SQLException {
 
     // changed max(total_revenue) to total_revenue // Not possible in normal to SELECT aggregate within inner query
```

Comment:
Why is this not possible?

Reply:
Fixed with https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1519992631, re-added.